src/handlers/logbook/getLogbook.js

```diff
@@ -26,5 +26,24 @@ export default async (req, res, next) => {
     return next(new HttpError(HttpStatusCode.NOT_FOUND, "Not Found"));
   }
 
+  let logbookDetails;
+  try {
+    logbookDetails = await db.LogbookDetails.findAll({
+      where: { logbook_id: logbook.id },
+      order: [["created_at", "DESC"]],
+    });
+  } catch (err) {
+    log.error(
+      `Error fetching logbook details ${logbookId}, error ${err.message}`
+    );
+    return next(
+      new HttpError(
+        HttpStatusCode.INTERNAL_SERVER_ERROR,
+        "Internal Server Error"
+      )
+    );
+  }
+  logbook.details = logbookDetails;
+
```

Comment:
https://sequelize.org/master/manual/eager-loading.html

Eager loading the relationship seems doable rather than manually running a second query, thoughts?

Reply:
Doing lazy loading, so it doesn't get loaded on each reference. See commit 3964b7